client/src/sendQuery.ts

```diff
@@ -39,8 +39,7 @@ export const buildSendQuery = async (input: {
     dataQuery,
     computeQuery,
     callback,
-    maxFeePerGas,
-    callbackGasLimit,
+    feeData,
```

Comment:
Nit, but would it make sense to order this like the sendQuery args?

Reply:
ack, good catch